kubectl-minio/cmd/helpers/helpers.go

```diff
@@ -148,3 +166,47 @@ func ToYaml(objs []runtime.Object) ([]string, error) {
 
 	return manifests, nil
 }
+
+// GetTable returns a formatted instance of the table
+func GetTable() *table.Table {
+	t := table.NewWriter(os.Stdout)
+	t.SetAutoWrapText(false)
+	t.SetHeaderAlignment(table.ALIGN_LEFT)
+	t.SetAlignment(table.ALIGN_LEFT)
+	return t
+}
+
+// DisableHelp disables the help command
+func DisableHelp(cmd *cobra.Command) *cobra.Command {
+	cmd.SetHelpCommand(&cobra.Command{
+		Use:    "no-help",
+		Hidden: true,
+	})
+	return cmd
+}
+
+// Ask user for Y/N input. Return true if response is "y"
+func Ask(label string) bool {
+	validate := func(input string) error {
+		s := strings.TrimSuffix(input, "\n")
+		s = strings.ToLower(s)
+		if strings.Compare(s, "n") != 0 && strings.Compare(s, "y") != 0 {
+			return errors.New("Please enter y/n")
+		}
+		return nil
+	}
+
+	prompt := promptui.Prompt{
+		Label:    label,
+		Validate: validate,
+	}
+	fmt.Println()
```

Comment:
```suggestion
	fmt.Println()
```

Is this needed?

Reply:
yeah just to add new line after the response. This is to ensure UI feels similar to other commands.